action.yml

```diff
@@ -9,7 +9,8 @@ inputs:
     required: true
   secrets:  # all secrets as json
     description: 'All the secrets serialized as json'
-    required: true
+    required: false
+    default: '${{toJson(secrets)}}'
```

Comment:
Smart 👍 For man vil jo aldrig gå glip af en god hemmelighed 😉 
Hvis man så alligevel ikke ønsker at give din action adgang til alle hemmeligheder, kan den vel overskrives tomt eller sættes til null?

Reply:
Det er ikke understøttet for secrets konteksten, hvilket giver meget god mening, da man i princippet kan hive alle secrets ud via en custom action, uden brugeren ved det. https://github.com/jnus/json-variables/issues/10